anta/input_models/services.py

```diff
@@ -29,3 +32,24 @@ def __str__(self) -> str:
         Server 10.0.0.1 (VRF: default, Priority: 1)
         """
         return f"Server {self.server_address} (VRF: {self.vrf}, Priority: {self.priority})"
+
+
+class ErrDisableReason(BaseModel):
+    """Model for an errdisable reason."""
+
+    model_config = ConfigDict(extra="forbid")
+    reason: ErrDisableReasons
+    """Type or name of the errdisable reason."""
+    status: Literal["Enabled", "Disabled"] = "Enabled"
```

Comment:
In the test, we previously checked that the status was enabled as static. Implementing this change (enabled: False) might introduce a breaking behavior.
@MaheshGSLAB @carl-baillargeon

Reply:
Ah yes I missed that. In that case, to avoid confusion let's keep the existing behaviour (no status/enabled knob) and we just check that it is "Enabled" in the code. If we get the request to explicitly check if recovery is "Disabled" for a reason later on, we'll add it. I doubt since by default they are all disabled in EOS.